admin-site/src/ServerURLstore.js

```diff
@@ -6,9 +6,10 @@
 
 import {writable} from 'svelte/store';
 
-export let urlStore = writable("https://staging.cohort.rocks/api/v2");
+export let urlStore = writable(window.location.protocol + '//' + window.location.host + '/api/v2');
```

Comment:
Can you find a way for this (important) constructed string to be declared in only one place?

Reply:
Is there a time/case where we would want to get server events from a different domain as the admin site ( local - staging vs local - local, or staging - staging)? If not we can take out the check local function all together and this string would only be declared once.